src/metrics/mod.rs

```diff
@@ -376,21 +343,37 @@ macro_rules! request {
     }
 }
 
-histogram!(
-    RESPONSE_LATENCY,
-    "response_latency",
-    "distribution of response latencies in nanoseconds."
-);
-
-histogram!(
-    SESSION_LIFECYCLE_REQUESTS,
-    "session_lifecycle_requests",
-    "distribution of requests per session lifecycle. incremented at time of session close."
-);
-
-histogram!(PUBSUB_LATENCY, "pubsub_latency");
-
-histogram!(PUBSUB_PUBLISH_LATENCY, "pubsub_publish_latency");
+#[metric(
+    name = RESPONSE_LATENCY_HISTOGRAM,
```

Comment:
Just to make sure, in the old code the macro expansion was `name = $name` so `metriken::metric` was getting just `RESPONSE_LATENCY', instead of `RESPONSE_LATENCY_HISTOGRAM`. Does this matter for any consumers?

Reply:
Previously name was the str literal passed to the macro. This str is now defined as its own const which previously was automatically defined by the macro. This PR shouldn't result in any change in the metric names.

Reply:
I was specifically referring to this bit, rather than the str for the name (or maybe we are talking about the same thing)?

```
($ident:ident, $name:tt, $description:tt) => {
        #[metriken::metric(
            name = $name,
```

I think in the old case the name was just the string (which wasn't appended with _HISTOGRAM)?

Reply:
The name previously was "response_latency", the ident for the histogram was RESPONSE_LATENCY with an automatically created RESPONSE_LATENCY_HISTOGRAM as a static str set to "response_latency" so we could index into the histogram snapshot with a name instead of having str literals in the metrics exposition bits.

Reply:
Ah, ok, fire away then.